docs/roles/ako-tenant.json

```diff
@@ -0,0 +1,250 @@
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ERRORPAGEBODY"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ALERTCONFIG"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ALERT"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ACTIONGROUPCONFIG"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ALERTSYSLOGCONFIG"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ALERTEMAILCONFIG"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_SNMPTRAPPROFILE"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_TRAFFIC_CAPTURE"
+        },
+        {
+            "type": "READ_ACCESS",
+            "resource": "PERMISSION_CLOUD"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_SERVICEENGINE"
+        },
+        {
+            "type": "WRITE_ACCESS",
+            "resource": "PERMISSION_SERVICEENGINEGROUP"
+        },
+        {
+            "type": "WRITE_ACCESS",
+            "resource": "PERMISSION_NETWORK"
+        },
+        {
+            "type": "WRITE_ACCESS",
+            "resource": "PERMISSION_VRFCONTEXT"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_USER_CREDENTIAL"
+        },
+        {
+            "type": "READ_ACCESS",
+            "resource": "PERMISSION_SYSTEMCONFIGURATION"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_CONTROLLER"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_REBOOT"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_UPGRADE"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_TECHSUPPORT"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_INTERNAL"
+        },
+        {
+            "type": "READ_ACCESS",
+            "resource": "PERMISSION_CONTROLLERSITE"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_IMAGE"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_USER"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_ROLE"
+        },
+        {
+            "type": "NO_ACCESS",
+            "resource": "PERMISSION_TENANT"
+        },
+        {
+            "type": "READ_ACCESS",
+            "resource": "PERMISSION_GSLB"
+        },
+        {
+            "type": "WRITE_ACCESS",
+            "resource": "PERMISSION_GSLBSERVICE"
+        },
+        {
+            "type": "WRITE_ACCESS",
```

Comment:
I think we have to give `READ_ACCESS` instead of `WRITE_ACCESS` here.

Reply:
updated with READ_ACCESS